process/accounts/accountsProcessor.go

```diff
@@ -168,6 +185,19 @@ func (ap *accountsProcessor) PrepareAccountsMapESDT(accounts []*data.AccountESDT
 	return accountsESDTMap
 }
 
+func convertStringToUint64(numberStr string) uint64 {
+	if numberStr == "" {
+		return 0
+	}
+
+	res, err := strconv.ParseUint(numberStr, 10, 64)
+	if err != nil {
+		return 0
```

Comment:
maybe add a log?

Reply:
Usually when converting a string to a number you also return the possible error you might encounter

Reply:
refactored this part. no longer need to convert nonce from string ( already is uint64)